content/en/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -460,14 +460,21 @@ An example flow:
       The containers in the Pod receive the TERM signal at different times and in an arbitrary
       order. If the order of shutdowns matters, consider using a `preStop` hook to synchronize.
       {{< /note >}}
-1. At the same time as the kubelet is starting graceful shutdown, the control plane removes that
-   shutting-down Pod from Endpoints (and, if enabled, EndpointSlice) objects where these represent
+
+{{< feature-state for_k8s_version="v1.22" state="beta" >}}
```

Comment:
I'm not sure if this feature state tag is appropriate here because this section also discusses behaviors that existed prior to the `EndpointSliceTerminatingCondition` feature.

Reply:
agree, removed